### DISCUSSION TITLE
Question about script filled attributes

### DISCUSSION THREAD

**Discussion by NOPKGer** [cwapi3d/cwapi3dpython#206] [Q&A] [answered]
Hey, 
I tried to use the function Check_axis from the Connector_axis_controller in the script filled attributes and then get the error that it is not found. 
However, the function runs via a standalone script. 
Do the script filled attributes access a different API version? And is there any documentation on which functions can be used?

**Comment by Brunner246** [comment, marked as answer]
Hi,

Not all functions are available for the script-filled attributes. 
We will publish a list of available functions with the next cwapi3d update.
